ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageSingle.test.ts

```diff
@@ -34,7 +36,11 @@ describe('Workload CVE Image Single page', () => {
         // Ensure the data in the table has settled
         cy.get(selectors.isUpdatingTable).should('not.exist');
 
-        cy.get('tbody tr td[data-label="Image"] a').first().click();
+        return cy.get('tbody tr td[data-label="Image"] a').then(([$imageLink]) => {
+            const imageName = $imageLink.innerText.replace('\n', '');
+            cy.wrap($imageLink).click();
+            return cy.get('h1').contains(imageName);
+        });
```

Comment:
Adjusting this function to query for the `h1` containing the image name lets us reliably determine that the visited page has been rendered.